launcher/tests/trusted-extensions.spec.ts

```diff
@@ -244,4 +251,41 @@ describe('Test Configuring of Trusted Extensions Auth Access:', () => {
     expect(spy).toHaveBeenCalledWith('  > add redhat.openshift');
     expect(spy).toHaveBeenCalledWith('  > failure to add [red hat.java] because of wrong identifier');
   });
+
+  test('should add only two extenions matching the regexp without case-sensitivity', async () => {
+    env.VSCODE_TRUSTED_EXTENSIONS = 'RedHat.Yaml,RedHat.OpenShift,red hat.java';
+
+    let savedProductJson;
+
+    Object.assign(fs, {
+      readFile: async (file: string) => {
+        if ('product.json' === file) {
+          return PRODUCT_JSON_SIMPLE;
+        }
+      },
+
+      writeFile: async (file: string, data: string) => {
+        if ('product.json' === file) {
+          savedProductJson = data;
+        }
+      },
+    });
+
+    const spy = jest.spyOn(console, 'log');
+
+    //test
+    const trust = new TrustedExtensions();
+    await trust.configure();
+
+    expect(savedProductJson).toBe(PRODUCT_JSON_TWO_EXTENSIONS_UPPERCASE);
+
+    expect(spy).toHaveBeenCalledWith('# Configuring Trusted Extensions...');
+    expect(spy).toHaveBeenCalledWith(
+      '  > env.VSCODE_TRUSTED_EXTENSIONS is set to [RedHat.Yaml,RedHat.OpenShift,red hat.java]'
+    );
+    expect(spy).toHaveBeenCalledWith('  > add RedHat.Yaml');
+    expect(spy).toHaveBeenCalledWith('  > add RedHat.OpenShift');
+    expect(spy).toHaveBeenCalledWith('  > failure to add [red hat.java] because of wrong identifier');
+  });
+
```

Comment:
Jobs are failing because of Code Style issues:

<img width="664" alt="image" src="https://github.com/che-incubator/che-code/assets/5676062/32762fea-bc74-4d9a-ae86-7db2dcb215a3">

Reply:
I ran the prettier command and fixed the code style

Reply:
thank you, I restarted PR checks, waiting for the result...